data/content/environment_variables.yaml

```
@@ -295,6 +295,14 @@ variables:
     A colon separated list of the pipeline's non-private team slugs.
   modifiable: false
   example: "deploy:ops:production"
+- name: BUILDKITE_PLUGIN_CONFIGURATION
+  desc: |
+    JSON string holding the current plugin's configuration (as opposed to all the plugin configurations in the `BUILDKITE_PLUGINS` environment variable).
```

Based on the other table entries, it looks like we include the article at the start of the description:
```suggestion
    A JSON string holding the current plugin's configuration (as opposed to all the plugin configurations in the `BUILDKITE_PLUGINS` environment variable).
```

---

Looking at the entry for `BUILDKITE_PLUGINS`, I notice it's described as a _JSON object_ rather than a _JSON string_. Should they both be the same, or is this one different?

---

BUILDKITE_PLUGINS _is_ a JSON object (that contains a list of plugins and their configs) :) A string is just a string.